scripts/check-helm-version.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/bash
+
+# exit when any command fails
+set -e
+
+function check_version_greater () {
+  local current=$1
+  local git_versions="${2:-0.0.0}"
```

Comment:
These aren't git versions so naming here is confusing. Perhaps `published_versions`?